templates/bot/js/notification-function-base/README.md

```diff
@@ -64,6 +64,8 @@ The default notifying logic is located in `bot/src/*Trigger.*s`, and you can als
 - Use `target.type` to distinguish different targets
 - Use `target.members()`, `target.channels()` to get more targets
 - Send your own card message
+- If your notification project is based on HTTP Triggers, send POST request to the `http://localhost/api/notification` with your favorite tools like postman.
```

Comment:
When I localdubug the endpoint is `http://localhost:3978/api/notification`, and when I run remote the endpoint is something like  `https://notification3d8c3fbot.azurewebsites.net/api/notification`.

I think it's better to follow [`adaptive-cards-notification`](https://github.com/OfficeDev/TeamsFx-Samples/tree/ga/adaptive-card-notification)'s README to use something like `https://<url>/api/notification` and then say: if you are localdebugging with ngrok the url could be `localhost:3978`; if you are using remote then the url could be your webapp/functionapp's url.